src/test-apps/happy/topologies/standalone/thread_wifi_ap_service.sh

```diff
@@ -23,6 +23,7 @@
 # - one access-point router onhub
 # - one service node instance cloud
 
+export PATH=$PATH:$TRAVIS_BUILD_DIR/src/test-apps/happy/bin
```

Comment:
`$TRAVIS_BUILD_DIR`  makes this specific to the Travis CI integration. please use something more general

Reply:
done